### PR TITLE
hotfix/1249 pub

### PR DESCRIPTION
**Pull request by gentee**
Changed pubkey for RFC

### CODE REVIEW

File: packages/api/login.go [hunk 1]
```diff
@@ -88,11 +88,14 @@ func login(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.En
 		logger.WithFields(log.Fields{"type": consts.EmptyObject}).Warning("state is empty, using 1 as a state")
 		ecosystemID = 1
 	}
-
+	dataKey = data.params[`pubkey`].([]byte)
+	if len(dataKey) == 65 && dataKey[0] == 4 {
```
Comment (potehinre) [inline packages/api/login.go]: move to separate function

File: packages/crypto/converter.go [hunk 2]
```diff
@@ -16,6 +16,9 @@ import (
 
 // Address gets int64 EGGAS address from the public key
 func Address(pubKey []byte) int64 {
+	if len(pubKey) == 65 && pubKey[0] == 4 {
```
Comment (potehinre) [inline packages/crypto/converter.go:19]: same stuff

**Review by potehinre — CHANGES_REQUESTED** [review]

File: packages/crypto/converter.go (hunk 2 above)
Comment (gentee) [reply packages/crypto/converter.go:19]: Fixed.

File: packages/api/login.go (hunk 1 above)
Comment (gentee) [reply packages/api/login.go]: Fixed

**Review by potehinre — APPROVED** [review]

**Pull request merged**